packages/jellyfish-api-core/__tests__/category/masternode/getMasternodeBlock.test.ts

```diff
@@ -0,0 +1,143 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { RpcApiError } from '../../../src'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getMasternodeBlocks', async () => {
+    const masternodeID = await client.masternode.createMasternode(await client.wallet.getNewAddress())
+
+    await container.generate(1)
+
+    const blocksResult = await client.masternode.getMasternodeBlocks({ id: masternodeID })
+
+    for (const value of Object.values(blocksResult)) {
+      expect(typeof value).toStrictEqual('string')
+      expect(value.length).toStrictEqual(64)
+    }
+  })
+
+  it('should getMasternodeBlocks with owner address', async () => {
+    const address = await client.wallet.getNewAddress()
+    await client.masternode.createMasternode(address)
+
+    await container.generate(1)
+
+    const blocksResult = await client.masternode.getMasternodeBlocks({ ownerAddress: address })
+
+    for (const value of Object.values(blocksResult)) {
+      expect(typeof value).toStrictEqual('string')
+      expect(value.length).toStrictEqual(64)
+    }
+  })
+
+  it('should getMasternodeBlocks with operator address', async () => {
+    const address = await client.wallet.getNewAddress()
+    await client.masternode.createMasternode(address)
+
+    await container.generate(1)
+
+    const blocksResult = await client.masternode.getMasternodeBlocks({ operatorAddress: address })
+
+    for (const value of Object.values(blocksResult)) {
+      expect(typeof value).toStrictEqual('string')
+      expect(value.length).toStrictEqual(64)
+    }
+  })
+
+  it('should getMasternodeBlocks with depth and owner address', async () => {
```

Comment:
Change the text to `should getMasternodeBlocks with owner address and depth`

Owner address = 1st param
Depth = 2nd param

Apply to other text too